chassis/sonic_platform/module.py

```diff
@@ -307,7 +309,16 @@ def set_admin_state(self, up):
         Returns:
             bool: True if the request has been successful, False if not
         """
-        return False
+        if self.get_type() == self.MODULE_TYPE_FABRIC:
+            if up == MODULE_ADMIN_DOWN:
+                logger.log_info("Power off {} module ...".format(self.module_name))
+                nokia_common._power_onoff_SFM(self.hw_slot,False)
+            else:
+                logger.log_info("Starting up chassis module {}".format(self.module_name))
```

Comment:
Change this log message to "Power on  {} module".  Please also test it, if it is already on,   set it on again should not trigger any bounce.